tests/ballerina-spec-conformance-tests/S05_simple_values/tests/int.bal

```diff
@@ -42,8 +42,8 @@ function testInt(int i1, int i2, int i3, int i4) {
     dataProvider: "decimalDataProvider"
 }
 function testOutOfRangeValueAsInt(decimal d) {
-    utils:assertPanic(function() returns int { return int.convert(d); },
-        "{ballerina}NumberConversionError", "invalid reason on out of range int value");
+    utils:assertPanic(function() returns int|error { return int.convert(d); },
+        "{ballerina}ConversionError", "invalid reason on out of range int value");
```

Comment:
ATM convert support string as well hence adding numeric conversion as error is not correct. Anyway string conversion is not part of convert. We can revert back to NumberConversionError once fixed that.